dandi/tests/test_upload.py

```diff
@@ -185,12 +185,10 @@ def test_upload_bids_invalid(
     mocker: MockerFixture, bids_dandiset_invalid: SampleDandiset
 ) -> None:
     iter_upload_spy = mocker.spy(LocalFileAsset, "iter_upload")
-    # Does it fail when it should fail?
-    with pytest.raises(RuntimeError):
-        bids_dandiset_invalid.upload(existing="forced")
+    bids_dandiset_invalid.upload(existing="force")
     iter_upload_spy.assert_not_called()
     # Does validation ignoring work?
-    bids_dandiset_invalid.upload(existing="forced", validation="ignore")
+    bids_dandiset_invalid.upload(existing="force", validation="ignore")
```

Comment:
How come all of that worked with a wrong value (`forced` instead of `force`) ? I would have assumed it was important to specify forcing in this test. Or it doesn't matter?

Reply:
I believe `existing` is only checked if there are any local assets being uploaded with the same path as a remote asset.  The BIDS sample Dandiset fixtures don't upload anything as part of their setup, so that code was never triggered.